README

```diff
@@ -219,6 +219,13 @@ existing ``dict``, modifying it using the API and serializing it back to a
 Development
 -----------
 
+Activate Virtual Environment:
```

Comment:
Could we also include a link to what virtual env is? Probably just point to http://docs.python-guide.org/en/latest/dev/virtualenvs/ or some other doc that describes what it is.